modules/world/CMakeLists.txt

```diff
@@ -52,6 +52,23 @@ if(NOT OPENCV_INITIAL_PASS)
   endforeach()
   message(STATUS "Processing WORLD modules... DONE")
   set(CMAKE_CURRENT_SOURCE_DIR "${OPENCV_MODULE_opencv_world_LOCATION}")
+
+  if(BUILD_opencv_cvv AND OPENCV_MODULE_opencv_cvv_IS_PART_OF_WORLD)
```

Comment:
To avoid mess, world module should not know modules details and handle modules specific code.

There is dedicated handlers in opencv_contrib.

Reply:
Under your reminder, I found that it was more appropriate to migrate the modifications to cor, but there is a small issue regarding the po of cmake.

The CMP0071 policy set in the cmakelist of cvv seems to be ineffective, and it seems that the setting of camkepolicy cannot cross the parent scope.

Should I write the settings for cmakepolicy to the cmakelist in the root directory? I have observed that many cmakepolicies are there.

Waiting for your reply. After receiving your suggestions, I will close this PR and make new modifications.

`CMP0071 did not take effect on world`
![image](https://github.com/opencv/opencv/assets/47711102/25f43301-81df-4daf-9911-72f526dd6918)